internal/rpcserver/rpcserverhelp.go

```diff
@@ -276,6 +276,7 @@ var helpDescsEnUS = map[string]string{
 
 	// GetBlockVerboseResult help.
 	"getblockverboseresult-hash":              "The hash of the block (same as provided)",
+	"getblockverboseresult-powhash":           "The PoW hash of the block (same as hash prior to DCP0011 activation)",
```

Comment:
For consistency with the only other mention of pow and since this is intended to be help, please make this "Proof-of-Work" instead of "PoW".

```suggestion
	"getblockverboseresult-powhash":           "The Proof-of-Work hash of the block (same as hash prior to DCP0011 activation)",
```